apps/src/acemode/annotationList.js

```diff
@@ -52,6 +52,15 @@ module.exports = {
   },
 
   /**
+   * Return the array of lintAnnotations
+   *
+   * @returns {!Array} annotations from most recent linting pass
+   */
+  getJSLintAnnotations: function () {
```

Comment:
Should this instead be `hasJSLintAnnotations` and return lintAnnotations.length, given that this is the only way it is used?